app/networkapi/people/models.py

```diff
@@ -92,6 +106,20 @@ class Person(models.Model):
         help_text='Do you want to feature this person?',
         default=False,
     )
+    publish_after = models.DateTimeField(
+        help_text='Make this person\'s profile visible only after '
+                  'this date and time (UTC)',
+        default=timezone.now,
```

Comment:
ok so this will not work because it is only calculated once and will be the same value for all model instances.
To make it default to the current time, just pass the `auto_now_add=True` param.